.travis.yml

```diff
@@ -21,6 +21,10 @@ addons:
     - texlive-latex-extra
     - dvipng
     - python-qt4
+    # - python3-qt4
+    # - python3-qt5
+    # - python3-pyside
```

Comment:
What's the deal here? We don't leave comments except in exceptional circumstances.

Reply:
That was a WIP. will remove. I was trying to install pyqt, old pyqt needed to be installed on the system. New pyqt can just be in a virtualenv.